qldbdriver/transaction.go

```diff
@@ -70,7 +70,27 @@ func (txn *transaction) execute(ctx context.Context, statement string, parameter
 	if err != nil {
 		return nil, err
 	}
-	return &Result{ctx, txn.communicator, txn.id, executeResult.FirstPage.Values, executeResult.FirstPage.NextPageToken, 0, txn.logger, nil, nil}, nil
+
+	// create IOUsage and copy the values returned in executeResult.ConsumedIOs
+	var ioUsage *IOUsage = nil
+	if executeResult.ConsumedIOs != nil {
+		readIOs := *executeResult.ConsumedIOs.ReadIOs
+		writeIOs := *executeResult.ConsumedIOs.WriteIOs
+		ioUsage = &IOUsage{
+			readIOs:  &readIOs,
+			writeIOs: &writeIOs,
+		}
+	}
+	// create TimingInformation and copy the values returned in executeResult.TimingInformation
+	var timingInfo *TimingInformation = nil
```

Comment:
As mentioned let's just never return nils.

Reply:
The default timingInfo is now:
```
var timingInfo = &TimingInformation{new(int64)}
```